runtime/mainnet/src/lib.rs

```diff
@@ -1040,4 +1132,198 @@ mod tests {
+				who_expected_balance
+			);
+		})
+	}
+
+	#[test]
+	fn test_fees_and_tip_split() {
+		new_test_ext().execute_with(|| {
+			let fee_amount = 10;
+			let fee =
+				<pallet_balances::Pallet<Runtime> as frame_support::traits::fungible::Balanced<
+					AccountId,
+				>>::issue(fee_amount);
+			let tip_amount = 20;
+			let tip =
+				<pallet_balances::Pallet<Runtime> as frame_support::traits::fungible::Balanced<
+					AccountId,
+				>>::issue(tip_amount);
+			let treasury_balance =
+				pallet_balances::Pallet::<Runtime>::free_balance(&TreasuryAccount::get());
+			let maintenance_balance =
+				pallet_balances::Pallet::<Runtime>::free_balance(&MaintenanceAccount::get());
+			DealWithFees::on_unbalanceds(vec![fee, tip].into_iter());
+
+			// Each to get 50%, total is 30 so 15 each.
+			assert_eq!(
+				pallet_balances::Pallet::<Runtime>::free_balance(&TreasuryAccount::get()),
+				treasury_balance + ((fee_amount + tip_amount) / 2)
+			);
+			assert_eq!(
+				pallet_balances::Pallet::<Runtime>::free_balance(&MaintenanceAccount::get()),
+				maintenance_balance + ((fee_amount + tip_amount) / 2)
+			);
+		});
+	}
+
+	#[test]
+	fn treasury_balance_converter_is_set() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::BalanceConverter>(),
+			TypeId::of::<UnityAssetBalanceConversion>(),
+		);
+	}
+
+	#[test]
+	fn treasury_block_number_provider_is_set() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::BlockNumberProvider>(),
+			TypeId::of::<System>(),
+		);
+	}
+
+	#[test]
+	fn treasury_burn_is_nothing() {
+		assert_eq!(TypeId::of::<<Runtime as pallet_treasury::Config>::Burn>(), TypeId::of::<()>(),);
+	}
+
+	#[test]
+	fn treasury_max_approvals_is_set() {
+		assert_eq!(<Runtime as pallet_treasury::Config>::MaxApprovals::get(), 100);
+	}
+
+	#[test]
+	fn treasury_pallet_id_is_set() {
+		assert_eq!(
+			<Runtime as pallet_treasury::Config>::PalletId::get().encode(),
+			PalletId(*b"treasury").encode()
+		);
+	}
+
+	#[test]
+	fn treasury_paymaster_is_correct_type() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::Paymaster>(),
+			TypeId::of::<TreasuryPaymaster<<Runtime as pallet_treasury::Config>::Currency>>(),
+		);
+	}
+
+	#[test]
+	fn treasury_payout_period_is_set() {
+		assert_eq!(<Runtime as pallet_treasury::Config>::PayoutPeriod::get(), 30 * DAYS);
+	}
+
+	#[test]
+	fn treasury_reject_origin_is_correct() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::RejectOrigin>(),
+			TypeId::of::<EnsureRoot<AccountId>>(),
+		);
+	}
+	#[test]
+	fn treasury_spend_funds_is_correct() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::SpendFunds>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn treasury_spend_origin_is_correct() {
+		assert_eq!(
+			TypeId::of::<<Runtime as pallet_treasury::Config>::SpendOrigin>(),
+			TypeId::of::<NeverEnsureOrigin<Balance>>(),
+		);
+	}
+
+	#[test]
+	fn treasury_spend_period_is_set() {
```

Comment:
Using ..._is_six_days instead would mean that anyone could browse the tests by name only and get a feeling for the configuration of the pallet before diving into the code.

The test names become a source of runtime documentation. It's a small change but can provide greater value (imo) with minimal effort.

Reply:
https://github.com/r0gue-io/pop-node/pull/428/commits/6d078eb3e6c15af2facb1aca85d5dd8e62694be1